docs/source/getting_started/dataset_creation/dataset_creation_azure_storage.rst

```diff
@@ -24,8 +24,26 @@ If you don't have a storage account or container yet follow the instructions `he
 Go to "Security + networking > Access keys". Copy the Key and store it in a secure location. Head to the next section to see how you can configure the Lightly dataset.
 
 
-Configuring a Lightly dataset to access the Azure storage
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Preparing your data
+^^^^^^^^^^^^^^^^^^^^^
+
+For Lightly to be able to create embeddings and extract metadata from your data, `lightly-magic` needs to be able to access your data. You can either download/sync your data from S3 or you can mount S3 as a drive. We recommend downloading your data from S3 as it makes the overall process faster.
```

Comment:
maybe also add a link to `:ref:`lightly-command-line-tool`.` similar as done in the glcoud tutorial